src/main/java/com/cburch/logisim/gui/start/Startup.java

```diff
@@ -95,7 +97,9 @@ public class Startup implements AWTEventListener {
   private String circuitToTest = null;
   private boolean exitAfterStartup = false;
   private boolean showSplash;
-  private File loadFile;
+
+  private List<Pair<File, String>> loadMems = new ArrayList<>();
```

Comment:
Please replace `loadMems` with something more self explanatory, like `memoryContentFiles` or so (maybe @davidhutchens would help). We do not need to save bytes on short and cryptic variable names.

Reply:
What about "memoriesToLoad" ?